package.json

```diff
@@ -35,6 +35,9 @@
     "watch-icons": "watchy -w src/icons -- npm run build-icons",
     "start": "eleventy --serve"
   },
+  "dependencies": {
+    "tailwindcss-interaction-variants": "^5.0.0"
```

Comment:
there is a duplicate declaration of this package in `devDependencies`, was this an accidental addition?

Reply:
(could this be why all the webkit additions to the css package?? 🤔)